app/src/main/java/com/example/bigowlapp/service/MemberScheduleAlarmReceiver.java

```diff
@@ -22,9 +24,13 @@
 
     @Override
     public void onReceive(Context context, Intent intent) {
-        // Execute Schedule activatedSchedule = getSchedule(intent); to get the schedule
-        // Run the location/Geofencing code
-        // activatedSchedule should only have UID, LONGITUDE, LATITUDE
+        Schedule activatedSchedule = getSchedule(intent);
+
+        ScheduledLocationTrackingManager locationTrackingManager = new ScheduledLocationTrackingManager(context);
+        // TODO: figure what to do with success/failure, or remove them
+        locationTrackingManager.addNewScheduledLocationToTrack(activatedSchedule)
+                .addOnSuccessListener(aVoid -> Log.e("BigOwl", "LOCATION TRACKING STARTED"))
+                .addOnFailureListener(e -> Log.e("BigOwl", e.getMessage()));
```

Comment:
I saw the comment, and I believe it's either remove them, or present a toast instead of logging.

Though, this should only be done before merging into staging; it could be useful for testing the PR code.

Reply:
- [ ] Completed?